dom.bs

```diff
@@ -1478,7 +1478,8 @@ steps:
 <ol>
  <li>Let <var>signal</var> be this object's {{PromiseController/signal}}.
  <li>Set <var>signal</var>'s [=PromiseSignal/aborted flag=].
- <li>[=Fire an event=] named <code>abort</code> at <var>signal</var>.</li>
+ <li>[=Fire an event=] named <code>abort</code> at <var>signal</var>.
+ <li>Run <var>signal</var>'s [=PromiseSignal/abort steps=].
```

Comment:
Inverted in https://github.com/whatwg/dom/pull/434/commits/8681c81ca74efa6a739eb499c95f2cece0f325f1.